zebedee-cms/src/test/java/com/github/onsdigital/zebedee/keyring/LegacyKeyringImplTest.java

```diff
@@ -99,279 +116,294 @@ public void setUp() throws Exception {
         when(collectionDescription.getId())
                 .thenReturn(TEST_COLLECTION_ID);
 
-        keyring = new LegacyKeyringImpl(sessionsService, cache, applicationKeys);
+        legacyKeyring = new LegacyKeyringImpl(sessionsService, users, keyringCache, applicationKeys);
     }
 
+
     @Test
-    public void testPopulateFromUser_userNull_shouldThrowException() {
+    public void testGet_userNull_shouldThrowException() {
         // Given user is null
 
-        // When populate from user is called
-        KeyringException ex = assertThrows(KeyringException.class, () -> keyring.populateFromUser(null));
+        // When get is called
+        KeyringException ex = assertThrows(KeyringException.class, () -> legacyKeyring.get(null, null));
 
         // Then an exception is thrown.
         assertThat(ex.getMessage(), equalTo(USER_NULL_ERR));
     }
 
     @Test
-    public void testPopulateFromUser_userEmailNull_shouldThrowException() {
+    public void testGet_userEmailNull_shouldThrowException() {
         // Given user email is null
         when(user.getEmail())
                 .thenReturn(null);
 
-        // When populate from user is called
-        KeyringException ex = assertThrows(KeyringException.class, () -> keyring.populateFromUser(user));
+        // When get is called
+        KeyringException ex = assertThrows(KeyringException.class, () -> legacyKeyring.get(user, null));
 
         // Then an exception is thrown.
         assertThat(ex.getMessage(), equalTo(EMAIL_EMPTY_ERR));
     }
 
     @Test
-    public void testPopulateFromUser_userEmailEmpty_shouldThrowException() {
+    public void testGet_userEmailEmpty_shouldThrowException() {
         // Given user email is empty
         when(user.getEmail())
                 .thenReturn("");
 
-        // When populate from user is called
-        KeyringException ex = assertThrows(KeyringException.class, () -> keyring.populateFromUser(user));
+        // When get is called
+        KeyringException ex = assertThrows(KeyringException.class, () -> legacyKeyring.get(user, null));
 
         // Then an exception is thrown.
         assertThat(ex.getMessage(), equalTo(EMAIL_EMPTY_ERR));
     }
 
     @Test
-    public void testPopulateFromUser_userKeyringNull_shouldThrowException() {
-        // Given user keyring is null
-        when(user.keyring())
-                .thenReturn(null);
+    public void testGet_collecitonNull_shouldThrowException() {
```

Comment:
There are a few instances of this typo in this file - I just highlighted one. 
```suggestion
    public void testGet_collectionNull_shouldThrowException() {
```